src/object_store/src/object/mod.rs

```diff
@@ -792,7 +792,10 @@ pub async fn build_remote_object_store(
 ) -> ObjectStoreImpl {
     match url {
         s3 if s3.starts_with("s3://") => {
-            if std::env::var("RW_USE_OPENDAL_FOR_S3").is_ok() {
+            if std::env::var("RW_USE_OPENDAL_FOR_S3")
+                .map(|v| v != "false")
+                .unwrap_or(true)
+            {
```

Comment:
Since we are not using a standard config here, let's print an info log when opendal is used so that we can know whether `RW_USE_OPENDAL_FOR_S3` works as expected in case we need to set it.